auth/jwt/transport.go

```diff
@@ -17,9 +17,9 @@ const (
 	bearerFormat string = "Bearer %s"
 )
 
-// ToHTTPContext moves JWT token from request header to context. Particularly
+// HTTPToContext moves a JWT from request header to context. Particularly
 // useful for servers.
-func ToHTTPContext() http.RequestFunc {
+func HTTPToContext() http.RequestFunc {
```

Comment:
Just curious, and this isn't part of the review for this PR, why have ToHTTPContext/HTTPToContext return the RequestFunc instead of be the RequestFunc? It always returns the same function value with no side effects.